src/Configuration.php

```diff
@@ -155,4 +161,24 @@ public function setValidator(Validator $validator): void
     {
         $this->validator = $validator;
     }
+
+    public function getValidationConstraints(): array
+    {
+        if ($this->constraints === null) {
+            return [];
+        }
+
+        return $this->constraints;
+    }
+
+    public function setValidationConstraints(array $constraints): void
```

Comment:
I'd rather have a variadic function for this, which would leave this validation up to PHP. Something like:

```php
public function setValidationConstraints(Constraint ...$constraints): void
{
    $this->constraints = $constraints;
}
```

Reply:
Very interesting. I never use variadic on this way. Thanks for the tip.